cli/src/command/finalize.rs

```diff
@@ -170,6 +184,7 @@ impl ExonumCommand for Finalize {
                 ..private_config.api
             },
             connect_list,
+            consensus_key: None, // We use it in creation connect list. We don't need it anymore.
```

Comment:
Is there a good reason _not_ to store it in the config (i.e. get rid of `Option`)?

Reply:
Also you can consider renaming it into `consensus_public_key`, as it is not clear when reading configs.

Reply:
The reason not to store is because of data duplication. We already have consensus key in *validator_keys* array.

Reply:
Is it bad? Right now we do not have a way to distinguish which private config corresponds to a public config.

Reply:
I agree with @vitvakatu that `consensus_public_key` would be a better name. Regarding duplication, I don't see it as a significant problem; for example, a node may be an auditor, in which case `consensus_public_key` is not duplicated. (As far as I understand, `exonum-cli` does not have first-class auditor support during blockchain creation, but this is a subject of future improvements.)